lib/manager/buildkite/extract.js

```diff
@@ -9,17 +9,24 @@ function extractPackageFile(content) {
   const deps = [];
   try {
     const lines = content.split('\n');
-    for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
-      const line = lines[lineNumber];
-      const plugins = line.match(/^\s*-?\s*plugins:\s*$/);
+    let pluginsSection = null;
+    let pluginsIndent = '';
+    for (let lineNumber = 1; lineNumber <= lines.length; lineNumber += 1) {
```

Comment:
Why we can't start at zero?

Reply:
We log these numbers and my point was to make it consistent with code listings in editors/github/etc